codespell_lib/data/dictionary.txt

```diff
@@ -4086,6 +4096,7 @@ desccription->description
 descencing->descending
 descendands->descendants
 descendend->descended, descendent, descendant,
+descendents->descendants
 descentences->descendants, descendents,
```

Comment:
Travis complains about the inconsistency here, that `descendents` is listed as an error and a correction

Reply:
Right. 
Line descendents->descendants deleted.